mmv1/templates/validator/resource_converter.go.erb

```diff
@@ -47,7 +47,7 @@ func resourceConverter<%= resource_name -%>() ResourceConverter {
     }
 }
 
-func Get<%= resource_name -%>CaiObject(d TerraformResourceData, config *transport_tpg.Config) ([]Asset, error) {
+func Get<%= resource_name -%>CaiObject(d tpgresource.TerraformResourceData, config *transport_tpg.Config) ([]Asset, error) {
```

Comment:
you'll need to add this import at the top of the file.

Reply:
Done.